actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -26,7 +26,42 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      class TagBuilder #:nodoc:
+        include TagHelper
 
+        VOID_ELEMENTS = %w(base br col embed hr img input keygen link meta
+        param source track wbr).to_set
+
+        VOID_ELEMENTS.merge(VOID_ELEMENTS.map(&:to_sym))
+
+        def initialize(view_context)
+          @view_context = view_context
+        end
+
+        private
+          def tag_string(name, content_or_options = nil, options_or_escape = nil, escape = true, &block)
```

Comment:
I love the idea, the only question remains what if someone wants to make HTML tag with `escape_attributes` attribute? :) I guess we can live with that.

One other non-obvious consequence - we won't be able to use double html attributes i.e.:
`tag("p", "class" => "show", :class => "elsewhere")`
Again - I guess we can live with that